rankers/MaxRanker/__init__.py

```diff
@@ -1,4 +1,7 @@
 from jina.executors.rankers import Chunk2DocRanker
+import warnings
```

Comment:
I would put it inside the `__init__` or `post_init` so that is not shown in every `jina` usage?

Reply:
nice catch! done